diff --git a/Configurations/15-ios.conf b/Configurations/15-ios.conf
index 1bb9f48d06..5bb2d5ab1e 100644
--- a/Configurations/15-ios.conf
+++ b/Configurations/15-ios.conf
@@ -59,4 +59,11 @@ my %targets = (
         CC               => "cc",
         cflags           => add("-isysroot \$(CROSS_TOP)/SDKs/\$(CROSS_SDK)"),
     },
+    "ios64sim-cross" => {
+        inherit_from     => [ "darwin-common", asm("no_asm") ],
+        cflags           => add("-arch x86_64 -DOPENSSL_NO_ASM -mios-version-min=7.0.0 -isysroot \$(CROSS_TOP)/SDKs/\$(CROSS_SDK) -fno-common"),
+        sys_id           => "iOS",
+        bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
+        perlasm_scheme   => "ios64",
+    },
 );
